src/NLog.Extensions.Logging/Logging/NLogProviderOptions.cs

```diff
@@ -56,6 +56,18 @@ public class NLogProviderOptions
 #endif
         public bool IncludeActivtyIdsWithBeginScope { get; set; }
 
+        /// <summary>
+        /// Reset the default Microsoft Logger Filter Options for the <see cref="NLogLoggerProvider"/>
+        /// </summary>
+        /// <remarks>This option affects the building of service configuration, so assigning it from appsettings.json has no effect (loaded after).</remarks>
+        public bool ResetLoggerProviderFilter { get; set; }
+
+        /// <summary>
+        /// Replace Microsoft ILoggerFactory with a pure <see cref="NLogLoggerFactory" />
+        /// </summary>
+        /// <remarks>This option affects the building of service configuration, so assigning it from appsettings.json has no effect (loaded after).</remarks>
+        public bool ReplaceLoggerFactory { get; set; }
```

Comment:
Proposal: UseNLogFactory

Reply:
I prefer replace, since it is replacing the existing LoggerFactory